fastestimator/dataset/data/german_ner.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+from pathlib import Path
+from typing import Optional, Tuple, Set, List
+
+import numpy as np
+import requests
+from sklearn.model_selection import train_test_split
+from tqdm import tqdm
+
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+
+
+def get_sentences_and_labels(path: str) -> Tuple[List[str], List[str], Set[str], Set[str]]:
+    """Combines tokens into sentences and create vocab set for train data and labels. For simplicity tokens with 'O'
+    entity are omitted.
+
+    Args:
+        path: Path to the downloaded dataset file
+
+    Returns:
+        (sentences, labels, train_vocab, label_vocab)
+    """
+    words, tags = [], []
+    word_vocab, label_vocab = set(), set()
+    sentences, labels = [], []
+    data = open(path)
+    for line in data:
+        if line[0] != '#':
+            line = line.split()
+            if len(line) > 2 and line[2] != 'O':
+                words.append(line[1])
+                tags.append(line[2])
+                word_vocab.add(line[1])
+                label_vocab.add(line[2])
+            else:
+                sentences.append(" ".join([s for s in words]))
+                labels.append([t for t in tags])
+                words.clear()
+                tags.clear()
+    sentences = list(filter(None, sentences))
+    labels = list(filter(None, labels))
+    return sentences[:10000], labels[:10000], word_vocab, label_vocab
+
+
+def load_data(root_dir: Optional[str] = None) -> Tuple[NumpyDataset, NumpyDataset, Set[str], Set[str]]:
+    """Dataset from GermEval 2014 contains 31,000 sentences corresponding to over 590,000 tokens from German wikipedia
```

Comment:
Details longer than a one line summary should go after a line break. see all of the other examples